### ISSUE TITLE
Skip symlinks in vendor folder on composer install/update.

### ISSUE THREAD

**Issue by Basster** [composer/composer#4125]
Related to #1549 I have a use case where I want to skip dependencies while "composer install/update".

We have an application, which relies on custom packages served through our private satis server, (as an example our application sdk). During the build process this is totally fine as it pulls the given version from the composer.json.

While developing I have the sdk on my machine as well, simlinking it in my applications vendor folder in the same structure composer would, if it is pulled on update/install. So changing the sdk in parallel behaves like its part of the project.

When I now want to update other dependencies I would love to have an option in the install/update command, to skip dependencies when they are either symlinks and/or marked as to be skipped.

Did I missed something on the documentation or would this make a good feature?

I'm going to try sending a PR for this, but I guess I need some time to get into the code.

**Comment by alcohol** [comment]
I think you linked the wrong issue, cause I don't see how these are related at all.

**Comment by alcohol** [comment]
If you want to update dependencies specifically, simply run `composer update vendor/package`.

**Comment by Seldaek** [comment]
This is rather a duplicate of #601 I'd say
